README.md

```diff
@@ -438,6 +465,8 @@ class SignupForm extends Component {
 }
 ```
 
+And violá, we can now control the submission process of the `<PaymentMethod />`. When combined with the various callbacks and some custom styling you can make the revops.js components fit into any workflow. 
```

Comment:
Is this filler documentation content? I like to keep the tone a little more humble.

Reply:
Ok, I toned down the claims a bit but still wanted to talk about a unified workflow. I think it is better now.